examples/slurm/epilogMoneo.sh

```diff
@@ -1,12 +1,6 @@
 #!/bin/bash
+log_file=/tmp/moneo-worker/id_change.log
 
-echo $SLURM_JOB_ID > /tmp/testfile.txt
-
-# The command below pipes stdout/stderr to files, Moneo will still update the respective moneo logs
-sudo -u <Your user> -H sh -c "cd  <Moneo DIR> &&  python3 moneo.py -j None -c /tmp/jobid_$SLURM_JOB_ID.txt >> /tmp/$SLURM_JOB_NAME.$SLURM_JOB_ID.Moneo.log 2>&1"
-
-rm /tmp/jobid_$SLURM_JOB_ID.txt
+sudo /tmp/moneo-worker/jobIdUpdate.sh  "None" > "$log_file" 2>&1
```

Comment:
same comment here - is this script not part of Moneo installation?

Reply:
This is the location Moneo is deployed to and runs out of. There is a possibility that the user does not have Moneo cloned or pre-installed (HPC images) on the compute nodes. The user would deploy from a remote machine using Moneo CLI which transfers necessary files to that location including this script.